app/components/vistk-scatterplot.js

```diff
@@ -16,6 +16,7 @@ export default Ember.Component.extend({
   scatter: computed('data.[]', 'dataType','eciValue','i18n.locale', function() {
     let eci = this.get('eciValue');
     let lang = this.get('i18n.locale') === 'en-col' ? 'en_EN': 'es_ES';
+    let format = function(value) { return numeral(value).format('0.0'); };
```

Comment:
I think we should increase the sig figures to 2 decimal points because the X axis does.
<img width="1061" alt="screen shot 2015-12-01 at 10 52 38 am" src="https://cloud.githubusercontent.com/assets/2522945/11505708/d4488eac-9819-11e5-866c-6dc10e0fcb07.png">